client/src/styles/Flex.jsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const Flex = styled.div`
+  flex-direction: ${({direction}) => direction || "row"};
```

Comment:
theme pointless, its row by defualt unless you say otherways. so basically - the default behavior is what you wrote, so its not needed